docker/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:20.04
+RUN \
+  # configure the "yeoman" user
+  groupadd yeoman && \
+  useradd yeoman -s /bin/bash -m -g yeoman -G sudo && \
+  echo 'yeoman:yeoman' |chpasswd && \
+  mkdir /home/yeoman/generator && \
+  export DEBIAN_FRONTEND=noninteractive && \
+  export TZ=Europe\Paris && \
```

Comment:
Why the TZ should be Paris? Do we require to have TZ specifically defined ?

I believe that the user can always provide that as a env argument:

```
docker run -rm -e TZ=Europe/Madrid -it myimage
```